### PR TITLE
Implemented suggest support

### PR DESCRIPTION
**Pull request by sbofirov**

### CODE REVIEW

File: src/Suggest/Suggest.php [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+     * @var string
+     */
+    private $text;
+
+    /**
+     * @var array
+     */
+    private $params;
+
+    public function __construct($name, $text, $params = [])
```
Comment (saimaz) [inline src/Suggest/Suggest.php]: We should stick to common behaviour and use `ParametersTrait`. You can leave $params in the constructor, but use trait function to set it. Take a look how e.g. https://github.com/ongr-io/ElasticsearchDSL/blob/master/src/Query/GeoShapeQuery.php#L36

**Comment by sbofirov** [comment]
@saimaz fixed.

**Comment by saimaz** [comment]
Docs are still missing for suggest usage. @sbofirov if you may add it would be great ;)

**Comment by sbofirov** [comment]
@saimaz will do

**Pull request merged**